include/titan/options.h

```diff
@@ -85,6 +85,11 @@ struct TitanCFOptions : public ColumnFamilyOptions {
   // Default: kNoCompression
   CompressionType blob_file_compression{kNoCompression};
 
+  // The compression options
+  //
+  // Default: disabled
+  CompressionOptions blob_file_compression_options;
```

Comment:
Default disable would be confusing, when `blob_file_compression` is not `kNoCompression`. We should ignore `blob_file_compression_options .enabled`, and mention it here.

Reply:
I will add more comments for this, and check if it handles this situation correctly